.neuro/live-s3.yml

```
@@ -0,0 +1,314 @@
+      export KEY=$(neuro blob statcredentials $CREDENTIALS | grep secret_access_key | awk '{print $2}')
+
+      echo "Starting label-studio"
+      python3 ${PROJECT}/label_studio/launch_ls.py \
+        --project_root ${PROJECT} \
+        --bucket ${BUCKET} \
+        --region_name ${REGION} \
+        --s3_endpoint ${ENDPOINT} \
+        --aws_access_key_id ${KEY_ID} \
+        --aws_secret_access_key ${KEY} \
+        -- \
+        start dogs --port ${LS_PORT} --no-browser
+
+
+      # Push labeling completions (label studio-specific files) and annotation results
+      neuro blob cp ${PROJECT}/data/result.json blob:${BUCKET_NAME}
+      neuro blob cp /label-studio/data/label_studio.sqlite3 blob:${BUCKET_NAME}
+
+      # Validate dataset:
+      neuro blob ls -r blob:${BUCKET_NAME}
+      echo "Total images: $(neuro blob ls -r blob:${BUCKET_NAME} | grep -oP '\w*\.\w+$' | wc -l)" # grep files only
+
+  deploy_inference_platform:
+    image: $[[ images.seldon.ref ]]
+    name: $[[ flow.title ]]-test-inference
+    preset: gpu-small
+    http_port: 5000
+    http_auth: False
+    life_span: 5h
+    detach: True
+    params:
+      run_id:
+        descr: MLFlow run ID, which model should be deployed
+      mlflow_storage:
+        descr: Storage path, where MLFlow server stores model binaries
+    volumes:
+      - ${{ params.mlflow_storage }}/0/${{ params.run_id }}/artifacts/model/data/model.h5:/storage/model.h5
+
+  locust:
+    image: locustio/locust:1.4.1
+    name: $[[ flow.title ]]-locust
+    http_port: 8080
+    http_auth: False
+    life_span: 1d
+    detach: True
+    browse: True
+    params:
+      endpoint_url:
+        default: ~
+        descr: |
+          Examples:
+          https://demo-oss-dogs-test-inference--<user-name>.jobs.<cluster-name>.org.neu.ro/api/v1.0/predictions - if model deployed as platform job
+          http://seldon.<cluster-name>.org.neu.ro/seldon/seldon/<model-name>-<model-stage>/api/v1.0/predictions -if model is deployed in Seldon
+    volumes:
+      - $[[ upload(volumes.src).ref_ro ]]
+      - $[[ upload(volumes.config).ref_ro ]]
+      - $[[ volumes.remote_dataset.ref_ro ]]
+    env:
+      DOG_IDS: "n02085936, n02088094"
+      IMGS_DIR: $[[ volumes.remote_dataset.mount ]]/images/Images/
+      PYTHONPATH: $[[ volumes.src.mount ]]/..
+    cmd: |
+      -f $[[ volumes.src.mount ]]/locust.py --web-port 8080 -H $[[ params.endpoint_url ]]
+
+  jupyter:
+    image: $[[ images.train.ref ]]
+    name: $[[ flow.title ]]-jupyter
+    preset: gpu-small
+    http_port: 8888
+    http_auth: False
+    browse: True
+    detach: False
+    volumes:
+      - "${{ upload(volumes.notebooks).ref_rw }}"
+      - ${{ params.mlflow_storage }}/0/${{ params.run_id }}/artifacts/model/data/model.h5:/storage/model.h5
+    env:
+      PYTHONPATH: /usr/project
+      PROJECT: /usr/project
+      EXPOSE_SSH: "yes"
+      DATA_PATH: ${{env.PROJECT}}/data/Images
```

`${{env.PROJECT}}` seems not to be working.. (at least in my case)
```suggestion
      DATA_PATH: /usr/project/data/Images
```